internal/provisioner/objects/service/service.go

```diff
@@ -369,42 +351,46 @@ func createService(ctx context.Context, cli client.Client, svc *corev1.Service)
 
 // updateContourServiceIfNeeded updates a Contour Service if current does not match desired.
 func updateContourServiceIfNeeded(ctx context.Context, cli client.Client, contour *model.Contour, current, desired *corev1.Service) error {
-	if labels.Exist(current, model.OwnerLabels(contour)) {
-		_, updated := equality.ClusterIPServiceChanged(current, desired)
-		if updated {
-			if err := cli.Update(ctx, desired); err != nil {
-				return fmt.Errorf("failed to update service %s/%s: %w", desired.Namespace, desired.Name, err)
-			}
-			return nil
+	if !labels.Exist(current, model.OwnerLabels(contour)) {
+		return nil
+	}
+	_, updated := equality.ClusterIPServiceChanged(current, desired)
+	if updated {
+		if err := cli.Update(ctx, desired); err != nil {
+			return fmt.Errorf("failed to update service %s/%s: %w", desired.Namespace, desired.Name, err)
 		}
 	}
 	return nil
+
```

Comment:
To be consistent with the change you made just above:
```suggestion
	_, updated := equality.ClusterIPServiceChanged(current, desired)
	if !updated {
	    return nil
	}

	
	if err := cli.Update(ctx, desired); err != nil {
		return fmt.Errorf("failed to update service %s/%s: %w", desired.Namespace, desired.Name, err)
	}
```